docs/sql-ref-syntax-qry-select-having.md

```diff
@@ -18,5 +18,101 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+The <code>HAVING</code> clause is used to filter the results produced by
+<code>GROUP BY</code> based on the specified condition. It is often used
```

Comment:
I guess you can link to the GROUP BY page?

Reply:
@huaxingao Good idea. +1